recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/domd-agl-image-copy-kernel/domd-agl-image-copy-kernel.bb

```diff
@@ -1,18 +0,0 @@
-LICENSE = "CLOSED"
```

Comment:
Commit message does not say anything about this file removal. Where did you put replacement?

Reply:
will be a separate patch